lua/telescope/pickers/entry_display.lua

```diff
@@ -3,6 +3,76 @@ local strings = require "plenary.strings"
 local entry_display = {}
 entry_display.truncate = strings.truncate
 
+---@tag telescope.pickers.entry_display
+
+---@brief [[
+--- Entry Display takes care of formatting the text displayed for each entry.
+--- This includes adding highlight colors.
```

Comment:
I dont know. I dont 100% like the way this is formulated. Entry Display doesnt take care of it, first `create` will create a function based on the input and returns that function. We then can use this function n times to return a string based on structured input.

And the highlighting part is just the second return value

Reply:
Its basically a `string.format` :laughing: